tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -172,8 +171,7 @@ def test_execute_create_cluster(self, mock_create_nodegroup, mock_create_cluster
         ]
 
         for (operator, parameters) in operator_under_test:
-            with self.subTest():
-                operator.execute({})
+            operator.execute({})
```

Comment:
Close builtin replacement to `unittest.TestCase.subTest` is parametrise.
You might have a look how I implemented this in initial PR: https://github.com/apache/airflow/pull/27970/files#r1034285155

Reply:
I actually didn't see that you were working on exactly this ^^ sorry for that.
Do you mind if I continue and just copy-paste your contribution to this file?

Reply:
As you wish, you could just grub whatever you think appropriate and help full.
But I didn't recommended copy-paste entire files because some changes might be add to this files.

Reply:
I added you as Co-Author of [921588e](https://github.com/apache/airflow/pull/28149/commits/921588e128f99bd2be702d54de285fe25e63f5a5), hopefully this will remain when merging the PR.